src/Resources/views/Form/form_admin_fields.html.twig

```diff
@@ -395,6 +401,8 @@ file that was distributed with this source code.
             {% if sonata_admin is defined and sonata_admin_enabled and sonata_admin.field_description.help|default(false) %}
                 <span class="help-block sonata-ba-field-help">{{ sonata_admin.field_description.help|trans({}, sonata_admin.field_description.translationDomain ?: admin.translationDomain)|raw }}</span>
             {% endif %}
+
+            {{ form_help(form) }}
```

Comment:
Here is the call @franmomu. GitHub messed up the code folding

Reply:
I saw it just after press Comment 😬